build-support/scripts/protobuf.sh

```diff
@@ -45,54 +44,29 @@ function main {
                 usage
                 return 0
                 ;;
-            --protoc-version )
-                protoc_version="$2"
-                shift 2
-                ;;
             --tools-only )
                 tools_only=1
                 shift
                 ;;
         esac
     done
 
-    if test -z "${protoc_version}"
-    then
-        protoc_version="$(make --no-print-directory print-PROTOC_VERSION)"
-        if test -z "${protoc_version}"
-        then
-            err_usage "ERROR: no proto-version specified and version could not be discovered"
-            return 1
-        fi
-    fi
-
-    # ensure the correct protoc compiler is installed
-    protoc_install "${protoc_version}"
-    if test -z "${protoc_bin}" ; then
-        exit 1
-    fi
-
     # ensure these tools are installed
     proto_tools_install
 
     if [[ -n $tools_only ]]; then
         return 0
     fi
 
-    # Compute some data from dependencies in non-local variables.
-    go mod download
```

Comment:
We may want to retain this step for CI. If the check generated protobuf thing passes without it then :shrug:

Reply:
I could put it back. Everything builds, lints and tests fine without it though.